p11-kit.yaml

```diff
@@ -63,14 +63,10 @@ subpackages:
     description: System trust module from p11-kit
     test:
       pipeline:
-        - uses: test/ldd-check
-          with:
-            packages: $(basename ${{targets.contextdir}})
+        - uses: test/tw/ldd-check
         - runs: |
             trust --help
-        - uses: test/ldd-check
-          with:
-            packages: p11-kit-trust
+        - uses: test/tw/ldd-check
```

Comment:
This is the second usage of the ldd-check test for this package.

Reply:
These were already there. I've removed the dupe.